core/constants.lua

```diff
@@ -0,0 +1,22 @@
+------------------------------
+-- Directions
+------------------------------
+NORTH     = 0
+EAST      = 1
+SOUTH     = 2
+WEST      = 3
+SOUTHWEST = 4
+SOUTHEAST = 5
+NORTHWEST = 6
+NORTHEAST = 7
+
```

Comment:
they're declared already, see line 1181 of this: https://github.com/otland/forgottenserver/blob/master/src/luascript.cpp